parser/src/test/java/com/linkedin/avroutil1/parser/avsc/AvscParserTest.java

```diff
@@ -72,6 +72,16 @@ public void testParseInvalidType2() throws Exception {
         Assert.assertTrue(result.getParseError() instanceof AvroSyntaxException);
     }
 
+    @Test
+    public void testRecordDefaults() throws Exception {
+        String avsc = TestUtil.load("schemas/RecordDefault.avsc");
+//        AvscParser parser = new AvscParser();
```

Comment:
you want to delete these commented out lines?

Reply:
also, want to assert that the values of defaults are picked up and are available at runtime?

Reply:
fixed